src/codeu/chat/client/commandline/Chat.java

```diff
@@ -147,6 +147,7 @@ public void invoke(List<String> args) {
           } else {
             //Print server info
             System.out.println("Version:" + info.version);
+            System.out.println("UpTime:" + info.upTime);
```

Comment:
This doens't seem to work. The code doesn't compile because `upTime` doesn't exist as a member of `info`. You'll want to calculate the length of time that the server has been up here.